03_letters/alanhillgemann/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	runeMap := make(map[rune]int)
+	runeSlice := []rune(s)
+	for i := 0; i <= len(runeSlice)-1; i++ {
+		runeMap[runeSlice[i]]++
+	}
+	return runeMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	stringSlice := make([]string, 0)
```

Comment:
Name things by what they're used for, not what they are. In this case, `keyValueStrings` might suffice.

(**Hot tip:** In VSCode, you can press <kbd>F2</kbd> to edit a name and all of its references.)

Reply:
Fixed. All variables renamed to be more descriptive